scripts/Deploy-MSBuild.ps1

```diff
@@ -73,11 +73,11 @@ $filesToCopyToBin = @(
     FileToCopy "$bootstrapBinDirectory\Microsoft.Managed.targets"
     FileToCopy "$bootstrapBinDirectory\Microsoft.Managed.Before.targets"
     FileToCopy "$bootstrapBinDirectory\Microsoft.Managed.After.targets"
-    FileToCopy "$bootstrapBinDirectory\Microsoft.Net.props"
-    FileToCopy "$bootstrapBinDirectory\Microsoft.NetFramework.CurrentVersion.props"
-    FileToCopy "$bootstrapBinDirectory\Microsoft.NetFramework.CurrentVersion.targets"
-    FileToCopy "$bootstrapBinDirectory\Microsoft.NetFramework.props"
-    FileToCopy "$bootstrapBinDirectory\Microsoft.NetFramework.targets"
+    FileToCopy "$bootstrapBinDirectory\Microsoft.NET.props"
```

Comment:
Ugh, this is terrible. The capitalization is different for netcoreapp and net framework. I'm not a powershell expert, but could you make it copy this file if Core and the other one if not?

Reply:
The change is needed because Linux file systems are case sensitive.
Windows filesystems are case insensitive. So this should continue to work on Windows as before.

Reply:
Right, but it won't necessarily work on macOS mono because that will look in the net framework package.